File: php/weebs.php
<?php

class weebs {

  public static function hug() {
    $text = file_get_contents("https://weebs.cf/random/hug");
    return $text;
  }

  public static function punch() {
    $text = file_get_contents("https://weebs.cf/random/punch");
    return $text;
  }

  public static function poke() {
    $text = file_get_contents("https://weebs.cf/random/poke");
    return $text;
  }

  public static function kiss() {
    $text = file_get_contents("https://weebs.cf/random/kiss");
    return $text;
  }

  public static function slap() {
    $text = file_get_contents("https://weebs.cf/random/slap");
    return $text;
  }

  public static function laugh() {
    $text = file_get_contents("https://weebs.cf/random/laugh");
    return $text;
  }
  

}

?>

File: python/weebs.py
import requests

def coin():
  return requests.get('https://weebs.cf/random/coin').text

def hug():
  return requests.get('https://weebs.cf/random/hug').text

def kiss():
  return requests.get('https://weebs.cf/random/kiss').text

def laugh():
  return requests.get('https://weebs.cf/random/laugh').text

def pat():
  return requests.get('https://weebs.cf/random/pat').text

def poke():
  return requests.get('https://weebs.cf/random/poke').text

def punch():
  return requests.get('https://weebs.cf/random/punch').text

def slap():
  return requests.get('https://weebs.cf/random/slap').text

def bite():
  return requests.get('https://weebs.cf/random/bite').text
  
def love():
  return requests.get('https://weebs.cf/random/love').text

def think():
  return requests.get('https://weebs.cf/random/think').text